project-templates/template-CONTRIBUTING.md

```diff
@@ -0,0 +1,20 @@
+How to contribute to Foo
```

Comment:
Does this mean we expect a separate CONTRIBUTING.md file, or can it be part of the README.md?